OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -480,6 +484,11 @@ public virtual void Stop()
 			StopSource();
 			AL10.alSourcei(Source, AL10.AL_BUFFER, 0);
 		}
+
+		public void SetLooping(bool looping)
+		{
+			AL10.alSourcei(Source, AL10.AL_LOOPING, looping ? AL10.AL_TRUE : AL10.AL_FALSE);
```

Comment:
If you merge bleed, you'll find this will need a guard against a sound that is `done` to avoid setting properties on a `Source` that has been reclaimed for use elsewhere.

Reply:
Rebased and added the bail out.